db/pre-re/item_db.conf

```diff
@@ -66942,7 +66952,7 @@ item_db: (
 	Job: 0x00000400
 	Upper: 8
 	BuyingStore: true
-	Script: <" //madoheal rand(300,400),0; ">
+	Script: <" if(checkmount() == MOUNT_MADO) itemheal rand(300,400), 0; ">
```

Comment:
3rd jobs are enabled in pre-re, what if server admin uses 3rd job in pre-renewal? Mechanics cannot use that item.